docs/versioned/admin/admin-overview.md

```diff
@@ -9,38 +9,60 @@ function: explanation
 
 This page provides guidance for administrators on how to manage Knative on an existing Kubernetes cluster.
 
+```mermaid
```

Comment:
I think we need at least a sentence or two of explanation before just dropping the diagram on folks.  (Particularly since it's a fairly big diagram.)

I think some color-coding would help substantially -- I'd suggest different colors for:

* Actors (developers and administrators)
* Developer-facing abstractions (Serving & Eventing)
* Make the "action" blocks have a transparent background when rendered.